pkg/image/importer/importer.go

```diff
@@ -335,6 +335,11 @@ func (isi *ImageStreamImporter) calculateImageSize(ctx gocontext.Context, repo d
 		size += desc.Size
 	}
 
+	if len(image.DockerImageConfig) > 0 && !blobSet.Has(image.DockerImageMetadata.ID) {
```

Comment:
> should we only do this if image.DockerImageManifestMediaType == schema2.MediaTypeManifest?

Both conditions should be equivalent:

    (len(image.DockerImageConfig) > 0) == (image.DockerImageManifestMediaType == schema2.MediaTypeManifest)

I'll use the first everywhere.